orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/MessageHandler.kt

```diff
@@ -37,6 +38,11 @@ interface MessageHandler<M : Message> : (Message) -> Unit {
       throw IllegalArgumentException("Unsupported message type ${message.javaClass.simpleName}")
     }
 
+  fun Collection<ExceptionHandler>.shouldRetry(ex: Exception, taskName: String?): ExceptionHandler.Response? {
```

Comment:
yeah, I think there's some stupid scenario where it couldn't find the task model too.